ruins-town/content/translation/ruins/polish.json

```diff
@@ -0,0 +1,202 @@
+{
+    "building.ruins-town.ruins.blacksmith.description": "Kuźnia wyposaża twoją armię w namioty pierwszej pomocy.",
```

Comment:
@ToRRent1812  Bad translation, you can buy ammunition carts in the blacksmith.

![obraz](https://github.com/vcmi-mods/ruins-town/assets/24764267/26b650d1-9a29-4bfd-b170-c296710e7f4f)
![obraz](https://github.com/vcmi-mods/ruins-town/assets/24764267/4b9d7760-2efe-4ac7-9e1d-a69a206a8ddd)

Reply:
oops